Developers/Development_Setup.rst

```diff
@@ -28,6 +28,7 @@ repository. The former is the core of coala, and the latter contains the set
 of standard bears. You can fork and clone the repositories from:
```

Comment:
also correction to the commit body: "The links to the x and y repositories appear on the same line." is more accurate